src/client/components/Modes/Modes.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import '../../styles/common.css';
+import ModesItem from '../ModesItems/modesItem';
+import { Image3d, ImageHeadSet, ImageSpeaker, ImageExtra } from 'src/images';
+
+class Modes extends Component {
```

Comment:
Better to use functional component instead of class component if you are not using component state here